frontend/libs/studio-components/src/components/StudioProperty/StudioProperty.stories.tsx

```diff
@@ -1,37 +1,24 @@
-import React from 'react';
+import type { ChangeEvent } from 'react';
+import React, { useEffect } from 'react';
 import type { Meta, StoryFn } from '@storybook/react';
-import {
-  StudioProperty,
-  type StudioPropertyButtonProps,
-  type StudioPropertyGroupProps,
-  type StudioPropertyFieldsetProps,
-} from './index';
-import { type StudioPropertyGroup } from './StudioPropertyGroup';
-
-type PreviewProps = {
-  withoutNegativeMargin: StudioPropertyGroupProps['withoutNegativeMargin'];
-  legend: StudioPropertyFieldsetProps['legend'];
-  buttons: StudioPropertyButtonProps[];
-};
-
-type PreviewStory = StoryFn<PreviewProps>;
-
-const ComposedPreviewComponent = ({ withoutNegativeMargin, legend, buttons }: PreviewProps) => {
-  return (
-    <StudioProperty.Group withoutNegativeMargin={withoutNegativeMargin}>
-      <StudioProperty.Fieldset legend={legend}>
-        {buttons.map((button) => (
-          <StudioProperty.Button key={button.property} {...button} />
-        ))}
-      </StudioProperty.Fieldset>
-    </StudioProperty.Group>
-  );
-};
+import { StudioProperty } from './index';
+import { ObjectUtils } from '@studio/pure-functions';
+import { StudioTextfield } from '../StudioTextfield';
+import { StudioButton } from '../StudioButton';
+import { XMarkIcon } from '@studio/icons';
 
 const meta: Meta = {
   title: 'Components/StudioProperty',
   component: ComposedPreviewComponent,
+  decorators: [
+    (Story) => (
+      <div style={{ width: 500, border: '1px solid #000', padding: 'var(--fds-spacing-5) 0' }}>
```

Comment:
Should we use a css module here?

Reply:
I chose this approach here since this component exists for documentation purposes only. Also, I think it's good to keep Storybook related stuff in one place, not only to make it easy to find, but also to avoid "noise" in the file system with stuff that's not used by the components themselves. However, you got a truly valid point. Our coding principles do apply here as well. I'll se if I can find a good way to structure this. Some of our components already have a `test-data` directory for complex test cases, so we can probably use a similar approach here.